test/TesterInternal/StorageTests/Relational/TestEnvironmentInvariant.cs

```diff
@@ -65,6 +65,11 @@ public class TestEnvironmentInvariant
                 {
                     StorageInvariant = AdoNetInvariants.InvariantNameMySql,
                     ConnectionString = "Server=127.0.0.1;Database=sys; Uid=root;Pwd=root;"
+                },
+                new StorageConnection
+                {
+                    StorageInvariant = AdoNetInvariants.InvariantNamePostgreSql,
+                    ConnectionString = "Server=127.0.0.1;Port=5432;Database=postgres;Integrated Security=true;Pooling=false;"
```

Comment:
@xclayl It looks like safest to remove the pooling feature. As you rightfuly pointed out, including this (without pgbouncer etc.) gives easily exceptions.